src/QueryDescription/index.js

```diff
@@ -23,7 +23,10 @@ import invariant from '../utils/common/invariant'
 
 import { type TableName, type ColumnName, columnName } from '../Schema'
 
+type $RE<Type> = $ReadOnly<$Exact<Type>>
```

Comment:
We should make it global type and use it everywhere to avoid confusion :)

Reply:
Agreed, though in a library we can't really make it "global", we'd need to import it anyway from a file. I could start a `src/types.js` file. A lot of types could be moved there too...

Reply:
👌 Are you going to do it in this PR?

Reply:
sure